.github/workflows/individual_evaluation.yml

```diff
@@ -36,36 +32,45 @@ jobs:
           echo $solo_reference
           echo "INPUT_INDIVIDUAL_EVALUATION=$solo_reference" >> $GITHUB_ENV
 
-      - name: Record and update benchmark animation
+      - name: Record and update benchmark animation and performance score
+        timeout-minutes: 10
```

Comment:
Do we want to enforce such a strict constraint on a template? If we mention it somewhere it might be fine, otherwise I think most organizers will break this limit quite easily. Without GPU, even fairly simple worlds (but more complex than just a thymio) might be beyond it, especially if cameras are involved.

Reply:
I think the limit is reasonable; if we look at the most complex benchmark we have, the highway driving benchmark took ~3 minutes to evaluate and the visual tracking took around 5 minutes.